example/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.1",
   "license": "MIT",
   "dependencies": {
-    "stats-webpack-plugin": "^0.2.1",
-    "webpack": "^1.9.11",
-    "webpack-dev-server": "^1.9.0"
+    "stats-webpack-plugin": "*",
```

Comment:
There is no release of this plugin that supports 2.2.0-rc.0 explicitly.  It currently requires 2.2.0-beta.  I honestly don't know what this even does, but nothing broke locally for me.

I'm guessing the steady-state of this might look like:

```json
{
  "stats-webpack-plugin": "^0.x.y", // whatever version is compatible
  "webpack": "^2.2",
  "webpack-dev-server": "^2.2"
}
```

(Note that I'm not super-familiar with node best practices for versioning—I'm trying to express "2.2 and above but not 3.0")